commentHandler.go

```diff
@@ -381,3 +386,7 @@ func removeMilestone(client *github.Client, ctx context.Context, URL string) err
 	}
 	return nil
 }
+
+func isDcoLabel(labelValue string) bool {
+	return strings.ToLower(labelValue) == "no-dco"
```

Comment:
Hey Richard I added the title in the test cases and Now I mention the user who requested to remove the label, but I'm confused about this comment? What do you mean by reconcile, maybe change the name?

Reply:
Apologies.  Simply, I'm thinking about a scenario where we might want to change the label in future - suppose we change to `no-sign-off` - we might change the create label occurrence but miss the occurrence in the remove label check which would mean that this feature would be broken.  

Would we be better off making both places use the same constant?

Reply:
Hey Richard thanks for the feedback! I replaced the "no-dco" string with constant that can be changed in case we want to change the label but it turns out, like you said, the label is used in lots of places in tests also.I was wondering is it a good idea to make separate PR which adds that constant and move from hard-coded "no-dco" to it or to make the change in the current one?

Reply:
Also tests values should be changed every time we change the `no-dco` label etc.